src/classes/webhooks/sendMail.js

```diff
@@ -0,0 +1,59 @@
+import nodemailer  from "nodemailer";
+import Action from '../action.js';
+import { DiscordWebhook } from "./discord.js";
+
+//Brugeren angiver mail
+//Angiver brugeren hvilken action der skal aktivere et mail send.
+//Bruge nodeMailer?
+//Vi sender fra vores Autoflow mail
+//Brugeren skal inputte deres mail,
+
+export class Nodemail extends Action {
+    constructor(mailTo, mailSubjectDefault = "Nothing to see", mailTextDefault = "Just checking in! :)") {
+        super();
+        this.mailTo;
+        this.mailSubjectDefault = mailSubjectDefault;
+        this.mailTextDefault = mailTextDefault;
+    }
+
+    execute(action, mailTo = this.mailTo, mailSubject = this.mailSubjectDefault, mailText = this.mailTextDefault){
+        
+        switch (action) {
+            case ("sendMail"):
+
+                //The sender authentication
+                var transporter = nodemailer.createTransport({
+                  service: 'gmail',
+                  auth: {
+                    user: process.env.MAIL_ADRESS,
+                    pass: process.env.MAIL_PASSWORD
+                  }
+                })
+                
+                //Who to send to and what to send
+                var mailOptions = {
+                    from: process.env.MAIL_ADRESS,
+                    to: mailTo,
+                    subject: mailSubject,
+                    text: mailText
+                }
+
+                transporter.sendMail(mailOptions, function(error, info) {
+                  if (error) {
+                    console.log(error);
+                  } else {
+                    console.log('Email sent: ' + info.response);
+                  }
+                });
+                break;
+            //If more, add
+            default:
+                console.log("Help TEAM, its wrong :(");
```

Comment:
kan i ikke omskrive det til [info] og så en eller andend seriøs besked

Reply:
I did the thing
Plz do the lgbtq